src/robusta/integrations/jira/client.py

```diff
@@ -205,23 +265,34 @@ def comment_issue(self, issue_id, text):
         response = self._call_jira_api(url, http_method=HttpMethod.POST, json=payload) or {}
         logging.debug(response)
 
-    def create_issue(self, issue_data, issue_attachments=None):
+    def create_issue(self, issue_data: dict, issue_attachments=None):
         endpoint = "issue"
         url = self._get_full_jira_url(endpoint)
-        payload = {
-            "update": {},
-            "fields": {
-                **issue_data,
-                "issuetype": {"id": str(self.default_issue_type_id)},
-                "project": {"id": str(self.default_project_id)},
-            },
-        }
-        logging.debug(f"Create issue with payload: {payload}")
-        response = self._call_jira_api(url, http_method=HttpMethod.POST, json=payload) or {}
-        logging.debug(response)
-        issue_id = response.get("id")
-        if issue_id and issue_attachments:
-            self.add_attachment(issue_id, issue_attachments)
+
+        priority_name = issue_data.get("priority", {}).get("name")
```

Comment:
Hi @ivankovnatsky 

This section changes the behavior for all users I think
I think now all existing users will get a different priority (I think it's a number), instead of the existing behavior

Can you please make it backward compatible, in a way that doesn't change the behavior to existing users?

Reply:
Hi Arik, thanks for the review. I understand that you reluctant to introduce
more changes as this stage, therefore I reverted to just handle my case, to be
able to pass custom user mappings. Otherwise, it should do current Robusta Jira
behavior.

WDYT?